tests/shared/nlu/training_data/test_entities_parser.py

```diff
@@ -137,3 +138,12 @@ def test_parse_training_example_with_entities():
     assert message.get("entities") == [
         {"start": 10, "end": 16, "value": "Berlin", "entity": "city"}
     ]
+
+
+def test_markdown_entity_regex_error_handling():
```

Comment:
there was no test for the error case? 🤯 Can we also add a test where it's a valid json but doesn't match the expected schema?

Reply:
sure